.travis.yml

```diff
@@ -37,13 +41,16 @@ before_script:
  - "if [ \"$BEHAT_TEST\" = \"\" ]; then php ~/travis-support/travis_setup.php --source `pwd` --target ~/builds/ss; fi"
  - "if [ \"$BEHAT_TEST\" = \"1\" ]; then php ~/travis-support/travis_setup.php --source `pwd` --target ~/builds/ss --require silverstripe/behat-extension; fi"
  - cd ~/builds/ss
+ - "if [ \"$CMS_TEST\" = \"1\" ]; then composer require --prefer-dist silverstripe/cms:$CORE_RELEASE.x-dev"
```

Comment:
Yeah I was aware of the `master-x.dev` issue, came to the same conclusion as you - we'd need to manually fix that. Didn't want to add it to `--require` in the bash lines above it, because it would always check out CMS with all its side effects.